docs/develop/convert-xml-to-json-manifest.md

```diff
@@ -0,0 +1,157 @@
+
+1. In the root of the project, open a command prompt or bash shell and run the following command. This converts the manifest and updates the package.json to specify current tooling packages. The new unified manifest is in the root of the project and the old XML manifest is in a backup.zip file. For details about this command, see [Office-Addin-Project](https://www.npmjs.com/package/office-addin-project).
+
+    ```command&nbsp;line
+    npx office-addin-project convert -m <relative-path-to-XML-manifest>
+    ``` 
+    
+1. Run `npm install`.
+1. To sideload the add-in, run `npm run start:desktop`. This command puts the unified manifest and the two image files into a zip file and sideloads it to the Office application. It also starts server in a separate NodeJS window to host the add-in files on localhost.
+
+### NodeJS and npm projects that aren't created with Yeoman Generator
+
+If you don't want to use the Teams Toolkit and your project wasn't created with the Office Yeoman generator, use the office-addin-manifest-converter tool.
+
+In the root of the project, open a command prompt or bash shell and run the following command. This command puts the unified manifest in a subfolder with the same name as the filename stem of the original XML manifest. For example, if the manifest is named MyManifest.xml, the unified manifest is created at .\MyManifest\MyManifest.json. For more details about this command, see [Office-Addin-Manifest-Converter](https://www.npmjs.com/package/office-addin-manifest-converter).
+
+```command&nbsp;line
+npx office-addin-manifest-converter convert -m <relative-path-to-XML-manifest>
+```
+
+Once you have the unified manifest created, there are two ways to create the zip file and sideload it. They are described in the next two subsections.
+
+#### Sideload with the Office-Addin-Debugging tool
+
+1. To sideload the add-in, run the following command. This command puts the unified manifest and two default icon image files into a zip file and sideloads it to the Office application. It also starts a server in a separate NodeJS window to host the add-in files on localhost. Note that you pass the path to the *unified manifest* that you created in the previous step. For more details about this command, see [Office-Addin-Debugging](https://www.npmjs.com/package/office-addin-debugging).
+
+    ```command&nbsp;line
+    npx office-addin-debugging start <relative-path-to-unified-manifest> desktop
+    ``` 
+
+1. When you use office-addin-debugging to start an add-in, *always stop the session with the following command*. Closing the server window doesn't reliably stop the server and closing the Office application doesn't reliably cause Office to unacquire the add-in.
+
+    ```command&nbsp;line
+    npx office-addin-debugging stop <relative-path-to-unified-manifest>
+    ``` 
+
+#### Sideload with the Teams Toolkit CLI
```

Comment:
For accessibility, style guide advises to define acronyms at first mention. I realize this is a nonstandard scenario though, because "Teams Toolkit CLI" is a product name, so I'll understand if you choose not to incorporate this suggestion. 
```suggestion
#### Sideload with the Teams Toolkit CLI (command-line interface) 
```